apps/meteor/app/livechat/server/api/v1/contact.ts

```diff
@@ -1,14 +1,18 @@
 import { LivechatCustomField, LivechatVisitors } from '@rocket.chat/models';
+import { isPOSTOmnichannelContactsProps } from '@rocket.chat/rest-typings';
 import { escapeRegExp } from '@rocket.chat/string-helpers';
 import { Match, check } from 'meteor/check';
 import { Meteor } from 'meteor/meteor';
 
 import { API } from '../../../../api/server';
-import { Contacts } from '../../lib/Contacts';
+import { Contacts, createContact } from '../../lib/Contacts';
 
+/**
+ * @deprecated to create a contact, use the omnichannel/contacts endpoint
+ */
 API.v1.addRoute(
 	'omnichannel/contact',
-	{ authRequired: true, permissionsRequired: ['view-l-room'] },
+	{ authRequired: true, permissionsRequired: ['view-l-room'], deprecationVersion: '8.0.0' },
```

Comment:
The `deprecationVersion` form is "deprecated" lol

Please use the object form
`deprecation: { version: xxx, alternatives: [''] }`